corehq/apps/smsbillables/migrations/0005_smsbillable_twilio_gateway_fee.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('smsbillables', '0004_bootstrap_smsgh_rates'),
+    ]
+
+    operations = [
```

Comment:
yup, as long as it's adding a nullable column with no default it shouldn't be too bad. I came across this which has some good info: http://blog.codacy.com/2015/05/14/how-to-update-large-tables-in-postgresql/

that said, since we're just adding a column, it might be worth it to have the person deploying this run this migration in a screen before the deploy happens to minimize the downtime